hs-bindgen/src/HsBindgen/C/Parser.hs

```diff
@@ -139,6 +142,16 @@ primType = either (const Nothing) aux . fromSimpleEnum
   Debugging
 -------------------------------------------------------------------------------}
 
+-- | An element in the @libclang@ AST
+data Element = Element {
+      elementDisplayName      :: Strict.ByteString
```

Comment:
Should these be strict fields?

Reply:
I generally don't bother with making things strict unless there is a good reason.

Reply:
I rather do opposite. Every field is strict, except if there is a reason for it to be lazy. Strict data, lazy functions.

Reply:
That works for me, though it gets annoying when it involves things like lists, `Maybe`, etc. etc.